server.js

```diff
@@ -54,6 +64,18 @@ app.use(serveStatic(STATIC_PATH, { index: false }));
 // FDK extension handler and API routes (extension launch routes)
 app.use("/", fdkExtension.fdkHandler);
 
+// Route to handle webhook events and process it.
+app.post('/api/product-delete', async function(req, res) {
```

Comment:
This should not be the generic one ? instead of `product-delete ` @jinalviranii

Reply:
Fixed.